local.builder.yml

```diff
@@ -1,8 +1,8 @@
-version: "2"
+version: "3"
 
 services:
   base:
-    image: node:12
+    image: node:14
```

Comment:
How confident are you that upgrading to Node 14 does not break anything?

Reply:
I'm fairly confident. This is a build-time only image, and the build doesn't fail. That said, I can revert this because we are now using a separate image to run Playwright.

Reply:
This has been reverted now; everything still runs fine.